### PR TITLE
rosdep: add key for Python3 scikit-image (skimage)

### PR DESCRIPTION
**Pull request by JulianKu**
Debian: https://packages.debian.org/search?lang=de&searchon=names&keywords=python3-skimage
Fedora: https://fedora.pkgs.org/31/fedora-x86_64/python3-scikit-image-0.14.2-2.fc31.x86_64.rpm.html
Gentoo: https://packages.gentoo.org/packages/sci-libs/scikits_image
PyPi: https://pypi.org/project/scikit-image/
Ubuntu: https://packages.ubuntu.com/search?lang=de&keywords=python3-skimage&searchon=names

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -5997,6 +5997,22 @@ python3-sip:
   fedora: [python3-sip]
   gentoo: [dev-python/sip]
   ubuntu: [python3-sip-dev]
+python3-skimage:
+  debian:
+    buster: [python3-skimage]
+    jessie: [python3-skimage]
+    stretch: [python3-skimage]
```
Comment (clalancette) [inline rosdep/python.yaml:6004]:
```
debian: [python3-skimage]
```

Should be sufficient and future-proof.

File: rosdep/python.yaml [hunk 2]
```diff
@@ -5997,6 +5997,22 @@ python3-sip:
...
+  gentoo: [sci-libs/scikits_image]
+  osx:
+    pip:
+      packages: [scikit-image]
+  ubuntu:
+    bionic: [python3-skimage]
+    eoan: [python3-skimage]
+    focal: [python3-skimage]
+    groovy: [python3-skimage]
+    xenial: [python3-skimage]
```
Comment (clalancette) [inline rosdep/python.yaml:6015]:
```
ubuntu: [python3-skimage]
```

Should work and be future-proof.

**Review by clalancette — CHANGES_REQUESTED** [review]

**Review by clalancette — APPROVED** [review]

**Comment by tfoote** [comment]
@JulianKu can you rebase this it conflicts with #24725

**Comment by JulianKu** [comment]
Sorry, somehow I messed up the rebase in between.
Fixed it though, so that merging should be possible now

**Review by jacobperron — APPROVED** [review]

**Pull request merged**